kafka-zookeeper/install

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+# install script used only in building the docker image, but not at runtime.
+# This uses relative path so that you can change the home dir without editing this file.
+# This also trims dependencies to only those used at runtime.
+set -eux
+
+echo "*** Installing Kafka and dependencies"
+APACHE_MIRROR=$(wget -qO- https://www.apache.org/dyn/closer.cgi\?as_json\=1 | sed -n '/preferred/s/.*"\(.*\)"/\1/gp')
+
+# Download scripts and config for Kafka and ZooKeeper, but not for Connect
+wget -qO- $APACHE_MIRROR/kafka/$KAFKA_VERSION/kafka_$SCALA_VERSION-$KAFKA_VERSION.tgz | tar xz \
+  --wildcards --strip=1 --exclude=connect* */bin/zookeeper-* */bin/kafka-* */config
+
+# Remove bash as our images don't have it, and it isn't required
+sed -i 's~#!/bin/bash~#!/bin/sh~g' bin/*sh
+
+# don't log ZK connections
+sed -i 's~log4j.logger.org.apache.zookeeper=INFO~log4j.logger.org.apache.zookeeper=WARN~g' config/log4j.properties
+# allow ruok command for testing ZK health
+echo 4lw.commands.whitelist=srvr, ruok >> config/zookeeper.properties
```

Comment:
we can remove this? basic zookeeper configuration are set later in the script.

Reply:
yep. this was a relic